bridge/core/api/form_data.d.ts

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright (C) 2022-present The WebF authors. All rights reserved.
+ */
+
+
+// This file is for generate binding code only.
+
+type FormDataPart={} // dummy code. Real one is introduced in C++ form_data_part.h, also blob_part.h
+export interface FormData {
+    new():FormData;
+    append(name: string, value: BlobPart, fileName?: string): void;
+    // This method name is a placeholder of **delete** method to avoid using C++ keyword
+    // and will be replaced to **delete** when installing in MemberInstaller::InstallFunctions.
+    form_data_delete(name: string): void;
```

Comment:
unknown API

Reply:
append: https://developer.mozilla.org/en-US/docs/Web/API/FormData/append
delete: https://developer.mozilla.org/en-US/docs/Web/API/FormData/delete

"delete" is renamed to "form_data_delete" here to avoid using C++ keyword, otherwise will cause a keyword conflict in generated C++ binding code.

It will be renamed back to "delete" when register into qjs , see: MemberInstaller::InstallFunctions